vm/Cargo.toml

```diff
@@ -26,6 +26,8 @@ rand = "0.8.5"
 rand_seeder = "0.2.2"
 ed25519-dalek = "1.0.1"
 itertools = "0.10.3"
+bech32 = "0.9"
```

Comment:
I don't think this dependency is needed.

Bech32 is more of a "front-end" thing.

Reply:
Done